packages/wonder-blocks-tooltip/src/components/tooltip.stories.js

```diff
@@ -48,6 +51,25 @@ const BaseTooltipExample = ({placement}: {|placement: Placement|}) => {
     );
 };
 
+export const TooltipOnButtons: StoryComponentType = () => {
+    return (
+        <View style={styles.centered}>
+            <View>
+                <Tooltip content={"This is a tooltip on a button."}>
+                    <Button disabled={false}>Example 1</Button>
+                </Tooltip>
+                <Strut size={Spacing.medium_16} />
+                <Tooltip
+                    content="This is a tooltip on a disabled button."
+                    placement="bottom"
+                >
+                    <Button disabled={true}>Example 2</Button>
+                </Tooltip>
+            </View>
+        </View>
+    );
+};
+
```

Comment:
We can probably skip publishing this story snapshot to Chromatic because the tooltip will only be displayed if the user focuses on the button.

```suggestion

TooltipOnButtons.parameters = {
    chromatic: {
        disableSnapshot: true,
    },
};

```

For more context, this is what you see in Chromatic and I don't think this visual test will need to check the tooltip itself: https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=61dcdd6e6e34b6003afb27ba